### DISCUSSION TITLE
Video Chat & Screensharing

### DISCUSSION THREAD

**Discussion by goll7535** [stoatchat/discussions#817] [Feature Suggestions]
It would be really nice to have a video chat and screen sharing functionality build in. So you could use it with direct chats and groups. Maybe take a look at jitsi. Its open-source and also has a self hosted option.
Plaese keep in mind that what we (the users and friends) ideally want is screenshare with audio, screensharing is an essential feature, but without the audio coming from the app being shared, it's often useless

**Comment by IAmTomahawkx** [comment]
duplicate of #196
